libraries/AP_HAL_ChibiOS/RCOutput.cpp

```diff
@@ -1660,7 +1660,7 @@ void RCOutput::dshot_send(pwm_group &group, rcout_timer_t cycle_start_us, rcout_
                 continue;
             }
 
-            const uint32_t chan_mask = (1U<<chan);
+            const uint32_t chan_mask = (1U<<(chan+chan_offset));
```

Comment:
This is not correct is it impacts the telemetry lower down. You should introduce a new variable or do the check actually in the if statement below.

Reply:
@andyp1per Thanks for the review, I have solved it now.